src/ec/packages/ec-component-ecl-compliance/ecl-compliance.html.twig

```diff
@@ -0,0 +1,52 @@
+{% spaceless %}
+
+{#
+  Parameters:
+  - "not-compliant" (boolean)
+  - "message" (string)
+  - "component" (string)
+  - "variant" (string)
+
+  Blocks:
+  - "complaints"
+  - "compliant"
+#}
+
+{# Internal properties #}
+
+{% set _success_message = '' %}
+{% set _identifier = component %}
+{% if variant %}
+  {% set _identifier = component ~ '--' ~ variant %}
+{% endif %}
+{% set _success_message = '<b>[' ~ _identifier ~ ']</b> - Your implementation is ecl compliant ;)' %}
```

Comment:
We already talked about the reason for the "success" message, i believe that if you test something you want to see an output from that, this is a debug tool, not anything fancy, therefore i don't really mind too much the way we output the messages, although we could clearly identify a better way.
Customising the markup of the component it's something i wouldn't want to have, for me the value is that you get the html output you would get from your implementation and then, separately, the comments by the "tool".
Anyway, i guess we could, potentially, use the results of the validation to set a class on the component root, for instance, but this would involve logic in the component template which is what i would want to avoid.

Reply:
Yes, I was checking the markup and it is true that adding a class or a wrapper around the component implies adding logic to the component code itself, which we want to avoid.

Reply:
ok let's start with this, we can always improve look and feel if necessary. we will see what it looks like globally and platform will perhaps give us some adjustment as well.